gravitee-apim-console-webui/package.json

```diff
@@ -182,5 +182,6 @@
     "test": "jest --detectOpenHandles",
     "test:auto": "jest --watch",
     "test:coverage": "node --max-old-space-size=7168 --expose-gc ./node_modules/.bin/jest --maxWorkers=3 --collectCoverage=true --logHeapUsage"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
```

Comment:
can you remove that
should you commit the yarn.lock ?
@Okhelifi can yo utell me ?

Reply:
I have no idea why yarn keeps adding packageManager. And there is no yarn.lock in base branch

Reply:
as per slack conversation: `package-lock.json` was updated